packages/web-components/src/presence-badge/index.ts

```diff
@@ -0,0 +1,32 @@
+import { customElement } from '@microsoft/fast-element';
+import { styles } from './presence-badge.styles';
+import { template } from './presence-badge.template';
+import { PresenceBadge } from './presence-badge';
+
+/**
+ * The Fluent Presence Badge Custom Element.
+ *
+ *
+ * @public
+ * @remarks
+ * HTML Element: \<fluent-presence-badge\>
+ *
+ */
+@customElement({
+  name: 'fluent-presence-badge',
+  styles,
+  template,
+})
+export class FluentPresenceBadge extends PresenceBadge {}
```

Comment:
You'll want to update this to use the `.compose({})` API from vNext which is also where you can set the default slotted content (SVG's).